### PR TITLE
Update Zyxel PMG3000-D20B, Nokia G-010s-A and BCM57810 pages (module compatibility and solder mod)

### PR DESCRIPTION
**Pull request by sodaws**
Hey,
here are the changes like discussed in #278. Please check if the links are working and the formatting is correct. I can't test this on my end right now.

Thanks

### CODE REVIEW

File: _ont/ont-nokia-g-010s-a.md [hunk 1]
```diff
@@ -287,6 +287,7 @@ omciMgr
 - [Usage GPON module SFP in Spain](https://forum.mikrotik.com/viewtopic.php?t=116364&start=300)
 - [Bypassing the HH3K up to 2.5Gbps using a BCM57810S NIC](https://www.dslreports.com/forum/r32230041-Internet-Bypassing-the-HH3K-up-to-2-5Gbps-using-a-BCM57810S-NIC)
 - [Nokia G-010S-A Pin 6 Iusse - Rsaxvc.net](https://rsaxvc.net/blog/2020/8/15/Nokia_G-010S-A_Pin_6_Issue.html)
+- [Might require solder mod in order to work with BCM57810S NIC](/_sfp_cage/broadcom-57810s.md#compatibility)
```
Comment (nanomad) [inline _ont/ont-nokia-g-010s-a.md:290]: Link not working, the PR will now build the preview site so you can check it :)

File: _sfp_cage/broadcom-57810s.md [hunk 2]
```diff
@@ -32,5 +32,17 @@ At this point, use the following command to set the speed to 2.5G:
...
+# Compatibility
+
+{% include alert.html content="This card requires a solder mod to work with some SFP ONTs." alert="Note"  icon="svg-warning" color="yellow" %}
+
+According to a [blog post](https://rsaxvc.net/blog/2020/8/15/Nokia_G-010S-A_Pin_6_Issue.html), some manufacturers designed their SFP ONTs in a way that doesn't 100% comply with the SFF specification. A solder mod like described in a [dslreports thread](https://www.dslreports.com/forum/r32529545-) might be required to get certain SFP modules to work with this card. This mod grounds `MOD_ABS PIN (6)`, which signals to the card that a module is inserted in the slot. This signals the card to send power to the SFP module.
+
+## Affected SFP Modules
+
+- [Zyxel PMG3000-D20B](/_ont/ont-zyxel-pmg3000-d20b.md) (see Github issue discussion in linked below)
+- [Nokia G-010S-A](/_ont/ont-nokia-g-010s-a.md)
```
Comment (nanomad) [inline _sfp_cage/broadcom-57810s.md:44]: Link not working

**Review by nanomad — DISMISSED** [review]

File: _ont/ont-nokia-g-010s-a.md (hunk 1 above)
Comment (sodaws) [reply _ont/ont-nokia-g-010s-a.md]: fixed

File: _sfp_cage/broadcom-57810s.md (hunk 2 above)
Comment (sodaws) [reply _sfp_cage/broadcom-57810s.md]: fixed

**Comment by sodaws** [comment]
Fixed the links and checked it on the preview website. Nice feature btw :)

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**